nemo_skills/prompt/config/deepseek/minif2f.yaml

```diff
@@ -12,21 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-executor: local
+few_shot_examples:
+  prefix: ""
+  template: |-
+    Complete the following Lean 4 code:
+      ‘‘‘lean4
+      import Mathlib
```

Comment:
Remove this and take directly from {header}. Include header field in the few-shots file. Also you don't need assistant end/begin, etc., instead of the format like in generic/math config